waku/common/envvar_serialization/reader.nim

```diff
@@ -72,7 +72,12 @@ proc readValue*[T](r: var EnvvarReader, value: var T) {.raises: [SerializationEr
   elif T is (seq or array):
     when uTypeIsPrimitives(T):
       let key = constructKey(r.prefix, r.key)
-      getValue(key, value)
+      try:
+        getValue(key, value)
+      except ValueError:
+        raise newException(
+          SerializationError, "Couldn't get value: " & getCurrentExceptionMsg()
+        )
```

Comment:
Is it out of the scope of the PR?

Reply:
For some reason we need it for the code to compile. After doing the changes, I get a compilation error saying that the proc `getValue` can raise a `ValueError` exception. Not sure why this error pops up with this change (why is that related to `uint16` or `uint32`).

But still, I guess that the extra fix is also an improvement and a correct usage of the proc